### PR TITLE
[ZT] Apply content strategy / clean up pages

### PR DESCRIPTION
**Pull request by maxvp**

### CODE REVIEW

**Comment by kodster28** [comment]
Looks like you got some broken links at https://github.com/cloudflare/cloudflare-docs/runs/6907676621?check_suite_focus=true (might've been introduced recently by someone else).

File: content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md [hunk 1]
```diff
@@ -16,76 +16,66 @@ Most of the parameters listed below are also configurable in the Zero Trust Dash
 
 ## Required for full Cloudflare One features
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md]:
```suggestion
## Required for full Cloudflare Zero Trust features
```

Comment (ranbel) [reply content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md]: We're in the process of changing the name from Zero Trust to Cloudflare One. For now, I'd leave it as Zero Trust.

File: content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md [hunk 2]
```diff
@@ -16,76 +16,66 @@ Most of the parameters listed below are also configurable in the Zero Trust Dash
...
 
-For the vast majority of Cloudflare Zero Trust features to work, you need to specify a team name. Examples of Cloudflare Zero Trust features depending on the team name are [HTTP policies](/cloudflare-one/policies/filtering/http-policies/), [Browser Isolation](/cloudflare-one/policies/browser-isolation/), and [device posture](/cloudflare-one/identity/devices/).
+For the majority of Cloudflare Zero Trust features to work, you need to specify a team name. Examples of Cloudflare Zero Trust features depending on the team name are [HTTP policies](/cloudflare-one/policies/filtering/http-policies/), [Browser Isolation](/cloudflare-one/policies/browser-isolation/), and [device posture](/cloudflare-one/identity/devices/).
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md]:
```suggestion
For the majority of Cloudflare Zero Trust features to work, you need to specify a team name. Examples of Cloudflare Zero Trust features which depend on the team name are [HTTP policies](/cloudflare-one/policies/filtering/http-policies/), [Browser Isolation](/cloudflare-one/policies/browser-isolation/), and [device posture](/cloudflare-one/identity/devices/).
```

File: content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md [hunk 3]
```diff
@@ -16,76 +16,66 @@ Most of the parameters listed below are also configurable in the Zero Trust Dash
...
 
 ### `organization`
 
-| Field          | Value Type |
-| -------------- | ---------- |
-| `organization` | string     |
+Instructs the client to register device with your organization. Registration requires authentication via an [IDP](/cloudflare-one/identity/idp-integration/) or [Service Auth](/cloudflare-one/identity/service-tokens/).
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md]:
```suggestion
Instructs the client to register the device with your organization. Registration requires authentication via an [IdP](/cloudflare-one/identity/idp-integration/) or [Service Auth](/cloudflare-one/identity/service-tokens/).
```

File: content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md [hunk 4]
```diff
@@ -16,76 +16,66 @@ Most of the parameters listed below are also configurable in the Zero Trust Dash
...
 
-**Description.** Instructs the client to register device with your organization. Registration requires authentication via an [IDP](/cloudflare-one/identity/idp-integration/) or [Service Auth](/cloudflare-one/identity/service-tokens/).
+**Value Type:** `string`
 
 **Value:** Your [team name](/cloudflare-one/glossary/#team-name).
 
-## Required field for DNS only policy enforcement
+## Required for DNS only policy enforcement
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md]:
```suggestion
## Required for DNS-only policy enforcement
```

File: content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md [hunk 5]
```diff
@@ -16,76 +16,66 @@ Most of the parameters listed below are also configurable in the Zero Trust Dash
...
 
 This field is only required to enforce DNS policies when deploying the client in DoH-only mode.
 
 ### `gateway_unique_id`
 
-| Field               | Value Type |
-| ------------------- | ---------- |
-| `gateway_unique_id` | string     |
+Instructs the client to direct all DNS queries to a specific policy location. This value is only necessary if deploying without a team name _or_ in an organization with multiple policy locations.
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md]:
```suggestion
Instructs the client to direct all DNS queries to a specific policy location. This value is only necessary if deploying without a team name or in an organization with multiple policy locations.
```

File: content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md [hunk 6]
```diff
@@ -16,76 +16,66 @@ Most of the parameters listed below are also configurable in the Zero Trust Dash
...
+Allows the user to control the connected state of the application (main toggle switch).
 
-**Description.** Allows the user to control the connected state of the application (main toggle switch).
+**Value Type:** `boolean`
 
 **Value:**
 
-- `false` \[default value] The user is able to turn switch on/off at their discretion. When the switch is off, the user will not have the ability to reach sites protected by Access that leverage certain device posture checks.
-- `true` The user is prevented from turning off the switch.
+- `false` &mdash; (default) The user is able to turn switch on/off at their discretion. When the switch is off, the user will not have the ability to reach sites protected by Access that leverage certain device posture checks.
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md]:
```suggestion
- `false` &mdash; (default) The user is able to turn the switch on/off at their discretion. When the switch is off, the user will not have the ability to reach sites protected by Access that leverage certain device posture checks.
```

File: content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md [hunk 7]
```diff
@@ -16,76 +16,66 @@ Most of the parameters listed below are also configurable in the Zero Trust Dash
...
-- `true` \[default value] Screen visible.
+- `false` &mdash; Screens hidden.
+- `true` &mdash; (default) Screens visible.
 
 ### `switch_locked`
 
-| Field           | Value Type |
-| --------------- | ---------- |
-| `switch_locked` | boolean    |
+Allows the user to control the connected state of the application (main toggle switch).
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md]:
```suggestion
Allows the user to turn off the WARP switch and disconnect the client.
```

File: content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md [hunk 8]
```diff
@@ -95,68 +85,60 @@ This parameter replaces the old `enabled` property, which can no longer be used
 
 ### `auto_connect`
 
-| Field          | Value Type |
-| -------------- | ---------- |
-| `auto_connect` | integer    |
+If switch has been turned off by user, the client will automatically turn itself back on after the specified number of minutes. We recommend keeping this set to a very low value &mdash; usually just enough time for a user to log in to hotel or airport WiFi.
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md:88]:
```suggestion
If switch has been turned off by user, the client will automatically turn itself back on after the specified number of minutes. We recommend keeping this set to a very low value &mdash usually just enough time for a user to log in to hotel or airport WiFi.
```

File: content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md [hunk 9]
```diff
@@ -95,68 +85,60 @@ This parameter replaces the old `enabled` property, which can no longer be used
...
 {{<Aside>}}
 This parameter replaces the old `enabled` property, which can no longer be used in conjunction with the new `switch_locked` and `auto_connect`. If you want to use these parameters, you must remove `enabled`.
 {{</Aside>}}
 
 ### `support_url`
 
-| Field         | Value Type |
-| ------------- | ---------- |
-| `support_url` | string     |
+When the WARP client is deployed via MDM, the in-app **Send Feedback** button is disabled by default. This parameter allows you to re-enable the button and direct it toward your organization.
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md]:
```suggestion
When the WARP client is deployed via MDM, the in-app **Send Feedback** button is disabled by default. This parameter allows you to re-enable the button and direct feedback towards your organization.
```

File: content/cloudflare-one/identity/idp-integration/google.md [hunk 10]
```diff
@@ -6,49 +6,26 @@ weight: 13
...
+You can integrate Google authentication with Cloudflare Access without a Google Workspace account. The integration allows any user with a Google account to log in (if the [Access policy](/cloudflare-one/policies/access/) allows them to reach the resource). Unlike the instructions for [Google Workspace](/cloudflare-one/identity/idp-integration/gsuite/), the steps below will not allow you to pull group membership information from a Google Workspace account.
 
-Please note that you don't need to be a Google Cloud Platform user to integrate Google Suite as an identity provider with Cloudflare Zero Trust. You will only need to open the Google Cloud Platform to access settings for your OIDC identity provider.
+You do not need to be a Google Cloud Platform user to integrate Google Suite as an identity provider with Cloudflare Zero Trust. You will only need to open the Google Cloud Platform to access settings for your OIDC identity provider.
 
-1.  Visit the Google Cloud Platform console. Create a new project.
+## Set up Google as an identity provider
 
-![Create Project](/cloudflare-one/static/documentation/identity/google/create-project.png)
```
Comment (ranbel) [inline content/cloudflare-one/identity/idp-integration/google.md:15]: If you remove an image and it's not being used elsewhere, you can go ahead and delete it from the `static/documentation` folder.

Comment (ranbel) [reply content/cloudflare-one/identity/idp-integration/google.md:15]: Oh, and also update this spreadsheet! https://docs.google.com/spreadsheets/d/1_dFVqJfVykGktS5O1JGupXs7rfCZ_GhKzBzjMMDf9Mw/edit#gid=1594008639

File: content/cloudflare-one/identity/idp-integration/google.md [hunk 11]
```diff
@@ -6,49 +6,26 @@ weight: 13
...
-1.  Name the project and click **Create**.
+1.  On the project home page, go to **APIs & Services** on the sidebar and select **Dashboard**.
 
-1.  On the project home page that loads, select **APIs & Services** from the sidebar and click **Dashboard**.
+1.  On the sidebar, go to **Credentials** and select **Configure Consent Screen** at the top of the page.
 
-1.  Go to **Credentials** and click **Configure Consent Screen** at the top of the page.
-
-![Click Consent](/cloudflare-one/static/documentation/identity/google/click-configure-consent.png)
+    ![Location of credential settings at the top of the Google Cloud Platform dashboard.](/cloudflare-one/static/documentation/identity/google/click-configure-consent.png)
```
Comment (ranbel) [inline content/cloudflare-one/identity/idp-integration/google.md:21]: We're tracking alt-text updates on this spreadsheet: https://docs.google.com/spreadsheets/d/1_dFVqJfVykGktS5O1JGupXs7rfCZ_GhKzBzjMMDf9Mw/edit#gid=1594008639
You don't have to update the "Current Alt-text" column, just check the "Done?" checkbox.

File: content/cloudflare-one/identity/idp-integration/google.md [hunk 12]
```diff
@@ -6,49 +6,26 @@ weight: 13
...
+1.  Name the application, add a support email, and input contact fields. Google Cloud Platform requires an email in your account.
+{{<Aside type="note">}}In the **Scopes** section, we recommend adding the `userinfo.email` scope. This is not required for the integration, but shows authenticating users what information is being gathered. You do not need to add test users.{{</Aside>}}
 
-You can review the summary information and return to the dashboard at the bottom of the page.
-
-![Summary](/cloudflare-one/static/documentation/identity/google/consent-screen-summary.png)
-
-1.  Return to the **APIs & Services** page and click _+ Create Credentials_. Select `OAuth client ID`.
-
-![Create OAuth](/cloudflare-one/static/documentation/identity/google/create-oauth.png)
```
Comment (ranbel) [inline content/cloudflare-one/identity/idp-integration/google.md:49]: Do you know if these screenshots are outdated? I know we're trying to reduce the # of screenshots, but they can be helpful for third-party integrations.

File: content/cloudflare-one/identity/idp-integration/google.md [hunk 13]
```diff
@@ -58,29 +35,23 @@ You can review the summary information and return to the dashboard at the bottom
...
+1.  Select **Save**.
 
-To test that your connection is working, navigate to **Authentication > Login methods** and click **Test** next to Google.
+## Next steps
 
-Your user identity should return.
+### Test your connection
 
-![Connection Works](/cloudflare-one/static/documentation/identity/google/connection-works.png)
+To test that your connection is working, go to **Authentication > Login methods** and select **Test** next to Google.
```
Comment (ranbel) [inline content/cloudflare-one/identity/idp-integration/google.md]:
```suggestion
To test that your connection is working, go to **Authentication** > **Login methods** and select **Test** next to Google.
```

File: content/cloudflare-one/identity/idp-integration/google.md (hunk 12 above)
Comment (maxvp) [reply content/cloudflare-one/identity/idp-integration/google.md:49]: Most of the screenshots didn't seem to convey much (e.g. screenshots of blank forms) and were poorly formatted/cropped. The OAuth one in particular could be useful, so I may edit it in the future.

**Review by ranbel — APPROVED** [review]

**Pull request merged**